lte/protos/enodebd.proto

```diff
@@ -93,6 +93,12 @@ message SingleEnodebStatus {
   string gps_latitude = 11;
   string fsm_state = 12;
   StatusProperty rf_tx_desired = 13;
+  string gps_altitude = 14;
+  string vendor = 15;
+  string model_name = 16;
+  StatusProperty rf_state = 17;
+  string sw_version = 18;
+  string uptime = 19;
```

Comment:
Please add some more details on the formats/valid values for uptime, and GPS altitude. The `proto` and swagger definitions should ideally be extremely well defined.